### PR TITLE
001 view job roles

### PR DESCRIPTION
**Pull request by mark-mcilroykainos**

### CODE REVIEW

**Comment by mark-mcilroykainos** [comment]
Hopefully sorted merge conflicts and branch is ready to be merged with main..

File: test/unit/services/JobRoleServiceTest.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            mock.onGet("http://localhost:8080/api/job-roles").reply(500);
+
+            try {
+                await getJobRoles();
+                throw new Error('Test failed - error was not thrown');
+            } catch (e) {
+                expect(e.message).to.equal('Failed to get Job Roles');
+            }
+        });
+
```
Comment (ntotten-kainos) [inline test/unit/services/JobRoleServiceTest.ts:46]: Super nit: whitespaces

**Review by ntotten-kainos — DISMISSED** [review]
LGTM : Just whitespaces might get caught by linter

File: src/services/JobRoleService.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import axios, { AxiosResponse } from "axios";
+import { JobRoleResponse } from "../models/JobRoleResponse";
+
+export const getJobRoles = async (): Promise<JobRoleResponse[]> => {
+    try {
+        const response: AxiosResponse = await axios.get("http://localhost:8080/api/job-roles"); // will need amended
```
Comment (shaunganley) [inline src/services/JobRoleService.ts]: this will work locally but not on AWS, please pass the base API URL in as an environment variable

**Review by shaunganley — CHANGES_REQUESTED** [review]
please add the coverage folder to your git ignore and delete them from here

**Review by shaunganley — APPROVED** [review]

**Review by ntotten-kainos — APPROVED** [review]

**Comment by ntotten-kainos** [comment]
LGTM

**Pull request merged**